<?xml version="1.0"?>
<launch>
	<arg name="robot" value="pg70"/>
	<arg name="can_device" default="can0"/>

	<!-- driver -->
	<include file="$(find cob_test_rigs)/launch/component.xml">
		<arg name="robot" value="$(arg robot)"/>
		<arg name="can_device" value="$(arg can_device)"/>
		<arg name="joint_names" value="['pg70_finger_left_joint']"/>
		<arg name="default_vel" value="0.4"/>
		<arg name="default_acc" value="1.0"/>
	</include>

</launch>
